docs/root/intro/arch_overview/load_balancing/locality_weight.rst

```diff
@@ -0,0 +1,69 @@
+.. _arch_overview_load_balancing_locality_weighted_lb:
+
+Locality weighted load balancing
+--------------------------------
+
+Another approach to determining how to weight assignments across different zones
+and geographical locations is by using explicit weights supplied via EDS in the
+:ref:`LocalityLbEndpoints <envoy_api_msg_endpoint.LocalityLbEndpoints>` message.
+This approach is mutually exclusive with the above zone aware routing, since in
```

Comment:
You'll probably need some rewording, since this is assuming a linear layout, e.g. "the above", "Another approach"..